modules/restful_token_auth/restful_token_auth.pages.inc

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Page callbacks for restful_token_auth module.
+ */
+
+/**
+ * Generates and shows an API key.
+ *
+ * @param \stdClass $account
+ *   A user account object.
+ *
+ * @return array
+ *   A render array.
+ */
+function restful_token_auth_get_credentials($account) {
+  $query = new EntityFieldQuery();
+  $result = $query
+    ->entityCondition('entity_type', 'restful_token_auth')
+    ->entityCondition('bundle', 'access_token')
+    ->propertyCondition('uid', $account->uid)
+    ->execute();
+
+  if (empty($result['restful_token_auth'])) {
+    return drupal_get_form('restful_token_auth_generate_token_form', $account->uid);
+  }
+  else {
+    $id = key($result['restful_token_auth']);
+    $auth_token = entity_load_single('restful_token_auth', $id);
+    return array(
+      '#theme' => 'restful_token_auth_user_key',
+      '#token' => $auth_token->token,
+    );
+  }
+}
+
+/**
+ * Form callback to generate an API token.
+ *
+ * @param array $form
+ *   The form array.
+ * @param array $form_state
+ *   The form state array.
+ * @param int $uid
+ *   The user identifier.
+ *
+ * @return array
+ *   The form as a render array.
+ */
+function restful_token_auth_generate_token_form($form, &$form_state, $uid) {
+  $form['description'] = array(
+    '#markup' => t('<p>You don\'t have an API key yet. Click below to generate one.</p>'),
```

Comment:
Let's avoid markup in the translation string. Maybe consider `#prefix` and `#suffix` for this.